### ISSUE TITLE
GO Wayback Machine

### ISSUE THREAD

**Issue by justinginnetti** [IFRCGo/go-web-app#1590]
**Is your feature request related to a problem? Please describe.**
At certain points in time we need to present not only what is on GO _right now_ (which we can capture with screenshots) but also _what was visible on GO at other points in time in the past_.

**Describe the solution you'd like**
As a pilot, initial example, we would like to show what was visible on GO -- and when -- with regard to Tropical Cyclone Chido from GDACS and from PDC. Here is a screenshot of what was available on 17.12.2024 at 15:00 CET for example:
![Image](https://github.com/user-attachments/assets/cac8a6b2-b366-4f3f-92d1-55d7f838f571)
 
We would like to go back in time and use the data from the logs to show the first instance of this event on GO and then each update that was visible, in each case indicating at which point in time this information was available. 

In the future, we will use this feature to visualise the comparisons between different impact-forecast models as well as between those models and CAP alerts disseminated via the IFRC AlertHub.

**Comment by tovari** [comment]
It would be good to replay the incoming data on on a local instance somehow.

**Comment by samshara** [comment]
@justinginnetti @tovari Are we considering a way to replay changes over time, allowing users to visualize the development of an event and navigate to specific points in time?

**Comment by justinginnetti** [comment]
@samshara , yes, this is a good way of describing the feature request. For the workshop in January that Luke mentioned, we just need the output/screen grabs; we do not need to push anything to staging then production. However, when we go to full implementation, we would want the capability you just described. Therefore, I've just added @mariam-yu to this ticket too.

**Comment by tovari** [comment]
I suggest to implement something simple that allows to replay the incoming data on a developer machine and make screenshots about the forecast phases on the risk watch UI. 
We don't need this functionality on the platform now.

**Comment by mariam-yu** [comment]
@justinginnetti There were really nice example of timelapse videos when we were looking into this with Thomas Roosli sometime back, but I can't seem to find them. Do you have them by chance? I could only find these, but there are global weather patterns and not for a specific event:
https://www.windy.com/?2024123018,53.645,-9.492,3
https://zoom.earth/

But essentially, having a little timelapse, playback would be great to show the historic of it (whenever we get to this).
